src/Base/Parameter.h

```diff
@@ -56,6 +56,7 @@ using PyObject = struct _object;
 
 #include "Handle.h"
 #include "Observer.h"
+#include "Color.h"

```

Comment:
And do forward declaring instead after making the parameter reference

Reply:
Then I cannot define the default because class is incomplete, I can skip the default (as it should be always specified explicitly anyway) but it would be inconsistent with rest of the API. What do you think would be the best move?